analyze_blog_posts.py

```
@@ -0,0 +1,188 @@
+#!/usr/bin/env python3
+import os
+import re
+import yaml
+from datetime import datetime
+import glob
```

Ruff (0.12.2)

`glob` imported but unused

Remove unused import: `glob`

(F401)